### ISSUE TITLE
Future of FreshRSS

### ISSUE THREAD

**Issue by marienfressinaud** [FreshRSS/FreshRSS#995]
Here is a ticket to discuss about future of FreshRSS.

**TL;DR:** I want to work on other projects, I have no time to spend on FreshRSS. So we need to finish what I started: [give FreshRSS' keys to community](https://github.com/FreshRSS/FreshRSS/issues/789).

These days I can't find motivation to work on FreshRSS. Many reasons for that but the main one is because I feel FreshRSS is nearly complete. Of course, I have a lot of ideas to improve FreshRSS but I can't find a way to not break everything and to rebuild everything :). I have no time for that and it's probably not a good idea. 

My idea is to give the keys to the community so you can continue the work (and restart FreshRSS if you want :p). Here's a list of tasks I have to finish before starting my new big (and probably awesome :D) personal project:
- [x] Buy a dedicated VPS for FreshRSS (there is no problem for me to pay for that, I still love and use FreshRSS :))
  - [x] Install web server
  - [x] Install mail server / mailing lists
  - [x] Migrate freshrss.org
  - [x] Migrate demo.freshrss.org
  - [x] Migrate doc.freshrss.org
  - [x] Migrate rss.freshrss.org
  - [x] Migrate update.freshrss.org
  - [x] Migrate dev.freshrss.org (dev version of freshrss.org)
  - [x] Migrate dev.update.freshrss.org (dev version of update.freshrss.org)
  - [x] Create dev.doc.freshrss.org (dev version of doc.freshrss.org)
  - [x] Migrate mailing lists
  - [ ] Open a tech zone
- [ ] Finish main documentation
  - [ ] Document Minz (see progress on https://github.com/FreshRSS/documentation/pull/11)
    - [x] Document Model section
    - [x] Document Controller section
    - [ ] Document View section
    - [ ] Document Request section
    - [ ] Document Dispatcher / FrontController section
    - [ ] Document Session section
    - [ ] Document Url section
    - [ ] Document Configuration section
    - [ ] Document Extension / ExtensionManager section
    - [ ] Document Log section
    - [ ] Document Translate section
  - [ ] Document application
    - [ ] Document Models
      - [ ] Entry
      - [ ] Feed
      - [ ] Category
      - [ ] Auth
      - [ ] Context
      - [ ] UserQuery
    - [ ] Document Controllers
      - [ ] Entry
      - [ ] Feed
      - [ ] Category
      - [ ] Auth
      - [ ] Subscription
      - [ ] Extension
      - [ ] Update
      - [ ] ImportExport
      - [ ] User
      - [ ] Configure
  - [ ] Document lib (third-party)
    - [ ] Document SimplePie / article fetching
    - [ ] Document favicon fetching
    - [ ] Document lib_rss
    - [ ] Document multi-themes
- [x] Document release (see https://github.com/FreshRSS/documentation/pull/12)
  - [x] Git process
  - [x] Services to update
  - [x] How to communicate
  - [x] Start new development version
- [x] Document auto-update process (see https://github.com/FreshRSS/documentation/pull/12 too)
  - [x] Write new script
  - [x] Update version.php
  - [x] Update dev.update.freshrss.org
  - [x] Test
  - [x] Update update.freshrss.org
- [x] Document documentation (yeah \o/) and publication (see https://github.com/FreshRSS/documentation/pull/13)
- [x] Document how-to write and publish blog post
- [ ] Explain how I see a potential FreshRSS redevelopment (it could be really awesome and well-designed :])
- [x] Release FreshRSS 1.2

Tell me if you think I forgot something. That's a lot of work so feel free to help me to accomplish this big last piece of work. I tried to determine tasks as atomic as possible in order to give me more motivation.

**Comment by dadall** [comment]
S'il te faut quelqu'un pour gérer le serveur, j'suis là ! :)

**Comment by marienfressinaud** [comment]
Oh c'est gentil ! :) Mais tu ne vas pas aimer : je prends toujours du CentOS :p Ceci dit j'hésite à prendre une Debian cette fois-ci mais j'ai peur de la migration depuis mon serveur actuel. On peut toujours en discuter par email si tu veux (mais à priori il n'y a pas grand chose à faire, je suis plutôt preneur de conseils sur l'aspect automatisation de l'installation)

**Comment by marcomrc** [comment]
I discover this script recently and it's great! :+1: (this is a motivation to work on FreshRSS! :+1: )
I hope the community can continue the work, but your support is important. 
In the next weeks I finish the italian translation and I make my contribution.

**Comment by marienfressinaud** [comment]
I close this ticket since almost all tasks were done except documentation. I'll reopen a ticket to suggest improvements on this specific subject.

**Comment by Frenzie** [comment]
PS The documentation ticket: https://github.com/FreshRSS/FreshRSS/issues/2491
